### PR TITLE
Run postprocess before consolidate in MZZ

### PR DESCRIPTION
**Pull request by peterm790**
This addresses issue https://github.com/fsspec/kerchunk/issues/176

### CODE REVIEW

**Comment by martindurant** [comment]
I see you called nb strip on this repo. We have a conflict between wanting rendered notebooks so that they are more interesting to see when linked from the examples age, versus cutting down further PR sizes for any future pages. @rsignell-usgs , what do you think?

**Comment by peterm790** [comment]
@martindurant or right I had setup nb-stripout to automatically run before committing, I will deactivate for the next commit.

Yes I am happy to have a look into the tests and why this failed

**Comment by martindurant** [comment]
Now has different failures :)  Unfortunately, changing the internal API often has knock-on effects like this, where tests are written to poke those internal methods. At a glance, it seems simple enough to change the test itself to match the new code for one, but I don't know about test_postprocess, probably needs more investigation.

**Comment by peterm790** [comment]
So this new commit passes all checks. I realised the initial way postprocess was written wasn't actually modifying `self.out`, which the test thankfully picked up on. Then I have made a small change to `test_combine` to call consolidate as a stand alone function rather than a module in `MultiZarrtoZarr`

File: kerchunk/combine.py [hunk 1]
```diff
@@ -404,13 +388,32 @@ def translate(self, filename=None, storage_options=None):
             self.store_coords()
         if 3 not in self.done:
             self.second_pass()
-        out = self.consolidate()
+        if 4 not in self.done:
+            if self.postprocess is not None:
+                self.out = self.postprocess(self.out)
```
Comment (martindurant) [inline kerchunk/combine.py:393]: Please alter the docstring for postprocess, to add its signature `postprocess(dict)->dict`.

**Pull request merged**